config/vm_repo_test.go

```diff
@@ -84,4 +84,84 @@ var _ = Describe("VMRepo", func() {
 			Expect(found).To(BeFalse())
 		})
 	})
+
+	Describe("FindCurrentIP", func() {
+		Context("when current IP is specified", func() {
+			BeforeEach(func() {
+				err := repo.UpdateCurrentIP("10.10.1.3")
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("returns current IP", func() {
+				ip, found, err := repo.FindCurrentIP()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(found).To(BeTrue())
+				Expect(ip).To(Equal("10.10.1.3"))
+			})
+		})
+
+		Context("when current IP is not specified", func() {
+			It("returns false", func() {
+				_, found, err := repo.FindCurrentIP()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(found).To(BeFalse())
+			})
+		})
+	})
+
+	Describe("UpdateCurrentIP", func() {
+		It("updates vm current IP", func() {
+			err := repo.UpdateCurrentIP("10.10.1.3")
+			Expect(err).ToNot(HaveOccurred())
+
+			deploymentState, err := deploymentStateService.Load()
+			Expect(err).ToNot(HaveOccurred())
+
+			expectedConfig := DeploymentState{
+				DirectorID: "fake-uuid-0",
+				CurrentIP:  "10.10.1.3",
+			}
+			Expect(deploymentState).To(Equal(expectedConfig))
+		})
```

Comment:
If no current IP is specified, it will delete the old vm and create a new one on Softlayer, just like other positive cases do.  
I'll add IP address validation in next patch.